### DISCUSSION TITLE
Codesandbox and Github not syncing

### DISCUSSION THREAD

**Discussion by lggomez18** [codesandbox/codesandbox-client#8204] [Q&A]
Hi, code sandbox stopped syncing 5 days ago. I've watched several videos and I can't get to the root of my issue, would someone please help me? **I am new in the coding environment, please use beginner-friendly wording :) Thanks. Here's my link. 

https://codesandbox.io/p/sandbox/website-trial-portfolio-gpngdv?file=%2Findex.html

**Comment by JamesACS** [comment]
Hello @lggomez18 - Sorry to hear about the trouble! 

Please could you clarify what you're expecting to occur here? This is a browser sandbox (so runs in your browser) so shouldn't be changing unless you make direct changes to the code.
